### DISCUSSION TITLE
Question:  How to convert `WinRT.IInspectable` to  `Windows.Foundation.Collections.IObservableVector`?

### DISCUSSION THREAD

**Discussion by shelllet** [dotnet/roslyn#55957] [Q&A]
![image](https://user-images.githubusercontent.com/16729017/131144993-1b2093cf-1fb1-4479-9899-a9c94921b9d8.png)

**Comment by huoyaoyuan** [comment]
This question should belong to microsoft/CsWinRT repo.

**Reply by shelllet** [reply]
yeah! it's my mistake.  i have no privilege to delete this topic.
